fusion-cli/docs/fusionrc.md

```diff
@@ -21,6 +21,30 @@ module.exports = {
 
 **Please note that custom Babel config is an unstable API and may not be supported in future releases.**
 
+## `jsExtPattern`
+
+ By default this is `/\.jsx?$/`
+
+ For example, this enables to handle typescript files with addition to Babel plugins/preset:
+
+ ```js
+module.exports = {
+  jsExtPattern: \[jt]sx?$\,
+  babel: {
+    presets: ["@babel/preset-typescript"],
+  }
+};
+```
+
+ ## `resolveExtensions`
+
+ By default this is webpack default witch is: `['.wasm', '.mjs', '.js', '.json']`
```

Comment:
Spelling: witch

Reply:
Grammar nitpick:

By default this is the Webpack default which is: